astroquery/sdss/core.py

```diff
@@ -56,10 +58,78 @@ class SDSSClass(BaseQuery):
     TIMEOUT = conf.timeout
 
     QUERY_URL = 'http://skyserver.sdss3.org/public/en/tools/search/x_sql.aspx'
+    XID_URL = 'http://skyserver.sdss.org/dr12/en/tools/crossid/x_crossid.aspx'
+
+    def query_crossid_async(self, coordinates, obj_names=None,
+                            photoobj_fields=None, specobj_fields=None,
+                            get_query_payload=False, timeout=TIMEOUT,
+                            radius=u.degree / 1800, output_format='csv'):
```

Comment:
Yeah, I was going to say it should be changed there too...

Just ping me when this is ready for final review & merging.